linux/hid.c

```diff
@@ -347,6 +359,9 @@ struct hid_usage_iterator {
  */
 static int get_next_hid_usage(const __u8 *report_descriptor, __u32 size, struct hid_usage_iterator *ctx, unsigned short *usage_page, unsigned short *usage)
 {
+	if (!report_descriptor || !pos || !usage)
```

Comment:
Note to self: this is nonsense :)

Reply:
So don't add it :)

There is no code-path where these arguments will not be valid.
And we have regular static analyzer checks to catch things like that.